miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -80,6 +81,14 @@ abstract class MiniApp internal constructor() {
         miniAppCustomPermission: MiniAppCustomPermission
     )
 
+    /**
+     * lists out the mini applications available with custom permissions in cache.
+     * @return [List<MiniAppInfo>] list of MiniApp what is downloaded and containing
+     * custom permissions data.
+     */
+    @Suppress("FunctionMaxLength")
+    abstract fun listDownloadedWithCustomPermissions(): List<MiniAppInfo>
```

Comment:
See the return type for this on iOS: https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/MiniApp.swift#L76

This should return both the `MiniAppInfo` and the `MiniAppCustomPermission` for each Mini App. So the return type could be like this: `List<Pair<MiniAppInfo, MiniAppCustomPermission>>`.

Reply:
Thanks, I'll update this part. Also, I'll keep the same name as listDownloadedWithCustomPermissions for the public interface.